src/spikeinterface/extractors/neoextractors/neuroscope.py

```diff
@@ -121,7 +113,7 @@ def __init__(
         exclude_shanks: Optional[list] = None,
         xml_file_path: OptionalPathType = None,
     ):
-        assert self.installed, self.installation_mesg
+        from lxml import etree as et
```

Comment:
```suggestion
        try:
            from lxml import etree as et
        except:
            raise ImportError(self.installation_mesg)
```
@h-mayorquin I thought we were still trying and raising an installation msg, but in the init. Am I wrong? See https://github.com/SpikeInterface/spikeinterface/blob/main/src/spikeinterface/extractors/cbin_ibl.py#L46-L49